logging/bin/getlogs.py

```diff
@@ -15,29 +15,34 @@
 
 def validate_input(dict):
     """Validate the arguments passed by the user to ensure script will function properly"""
-    if args['create-auth']:
+    if dict['create-auth']:
         if (type(dict['create-auth'] == list)):
             dict['create-auth'] = " ".join(dict['create-auth'])
-        x = open(args['create-auth'], "w")
+        x = open(dict['create-auth'], "w")
         x.write('{"username": " " , "password": " " , "host": " ", "port": " "}')
         print("\nAuth File created.\n")
 
     if dict['auth-file']:
         if (type(dict['auth-file'] == list)):
             dict['auth-file'] = " ".join(dict['auth-file'])
         try: 
-            a = open(args['auth-file'], 'r')
+            a = open(dict['auth-file'], 'r')
             authDict = json.loads(a.read())
-            args['userName'] = authDict['username']
-            args['password'] = authDict['password']
-            args['host'] = authDict['host']
-            args['port'] = authDict['port']
+            dict['userName'] = authDict['username']
+            dict['password'] = authDict['password']
+            dict['host'] = authDict['host']
+            dict['port'] = authDict['port']
         except Exception as e:
             print("There was a problem with the authentication file. Please ensure that the values are in dictionary format with keys: 'username', 'password', 'host', 'port'")
             print(e)
             exit()
 
-    
+    """Ensure maxrows is less than 10000"""
+
+    if dict['maxInt'] > 10000:
```

Comment:
Although its fairly clear whats happening here, lets get rid of the magic number here and define a constant. Could be useful if in the future we want to make it configurable but really just helps with maintainability/readability with the code 

```python
MAX_ROWS = 1000

if dict['maxInt'] > MAX_ROWS:
```